platform/ext/target/nordic_nrf/Native_Driver/spu.c

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "spu.h"
+#include <hal/nrf_spu.h>
+
+/* Platform-specific configuration */
+#define FLASH_SECURE_ATTRIBUTION_REGION_SIZE 0x8000
+#define SRAM_SECURE_ATTRIBUTION_REGION_SIZE  0x2000
+
+#define NUM_FLASH_SECURE_ATTRIBUTION_REGIONS 32
+#define NUM_SRAM_SECURE_ATTRIBUTION_REGIONS  32
+
+/* Minimum size of Non-Secure Callable regions. */
+#define FLASH_NSC_MIN_SIZE 32
+
+/* Maximum size of Non-Secure Callable regions. */
+#define FLASH_NSC_MAX_SIZE 4096
+
+
+/* Convenience macros for SPU attributes */
+#define FLASH_EXEC                                                             \
```

Comment:
Why not just use `NRF_SPU_MEM_PERM_*` values?

Reply:
Sure, I just did not know these macros exist. Will use them, thanks @anangl

Reply:
This, together with the comment above, @anangl @oyvindronningstad allows me to get rid of all locally-defined convenience macros for SPU permissions